modules/dxp/apps/commerce/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/DiagramTable/MappedProductRow.js

```diff
@@ -51,6 +52,13 @@ export default function MappedProductRow({
 			{!isAdmin && (
 				<ClayTable.Cell>
 					<ClayCheckbox
+						aria-label={sub(
+							Liferay.Language.get('select-sku-x,x'),
```

Comment:
I think if you check the other language keys we have the more common pattern would be `select-sku-x-x`.